lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -158,10 +158,10 @@ bool IsArguments(ParseNode *pnode)
             pnode = pnode->sxTri.pnode3;
             break;
 
-        //
-        // Cases where we don't check for "arguments" yet.
-        // Assume that they might have it. Disable the optimization is such scenarios
-        //
+            //
+            // Cases where we don't check for "arguments" yet.
+            // Assume that they might have it. Disable the optimization is such scenarios
+            //
```

Comment:
I didn't mean some indentation changes - VS have done that for me. If you have concerns regarding that let me know.